packages/grafana-runtime/src/services/ScopesDashboardsContext.ts

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext, useMemo } from 'react';
+import { useObservable } from 'react-use';
+
+import { ScopesDashboardsService, ScopesDashboardsServiceState } from './ScopesDashboardsService';
+import { getScopesDashboardsService } from './scopes';
+
+export const ScopesDashboardsContext = createContext<ScopesDashboardsService>(getScopesDashboardsService());
+
+export function useScopesDashboards() {
+  const context = useContext(ScopesDashboardsContext);
+  const state: ScopesDashboardsServiceState | undefined = useObservable(context.stateObservable, context.state);
+
+  return useMemo(() => {
+    return {
+      state,
+      changeSearchQuery: context.changeSearchQuery.bind(context),
+      clearSearchQuery: context.clearSearchQuery.bind(context),
+      togglePanel: context.togglePanel.bind(context),
+      updateFolder: context.updateFolder.bind(context),
+    };
+  }, [context, state]);
+}
```

Comment:
what are the use cases here for plugins to use these functions? It's not really clear. 

I could see use case of getting dashboards that match current scope or a specific scope and somehow showing those dashboards in the app outside the dashboards list pane. But not clear what the use case is here. Is it an action in the scene app should open the dashboard list pane with a specific search query?